OpenRA.Mods.Common/Widgets/Logic/MissionBrowserLogic.cs

```diff
@@ -319,6 +339,44 @@ void RebuildOptions()
 					.Where(o => o.IsVisible)
 					.OrderBy(o => o.DisplayOrder).ToArray();
 
+			tabsVisible = !allOptions.All(o => o.Id == "difficulty" || o.Id == "gamespeed");
+			if (tabsVisible)
+				BuildLargeOptions(allOptions);
```

Comment:
Large and Small feel odd IMO. How about `BuildOptionsTab` and `BuildOptionsButtons`?

Reply:
That feels odd as well, maybe minified vs regular (no name)?